service/controller/appcatalog/v1/key/key.go

```diff
@@ -0,0 +1,37 @@
+package key
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	versionBundleAnnotation = "giantswarm.io/version-bundle"
+)
+
+func AppCatalogTitle(customObject v1alpha1.AppCatalog) string {
+	return customObject.Spec.Title
+}
+
+// ToCustomObject converts value to v1alpha1.App and returns it or error
```

Comment:
We wanted to start aligning more with upstream. This would then be `ToCustomResource` and `customResource` respectively.

Reply:
That's nice. Lets do that. I'll also change the release resource in chart-operator.